packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -1,11 +1,15 @@
 import {
     assignFilteredGlobalDescriptorsFromPropertyDescriptorMap,
+    CallableEvaluate,
     createConnector,
     createMembraneMarshall,
     getFilteredGlobalOwnKeys,
     linkIntrinsics,
     DistortionCallback,
     Instrumentation,
+    Pointer,
```

Comment:
this might be problematic... to leak this into this file...

Reply:
It's just temporary while we try to troubleshoot things